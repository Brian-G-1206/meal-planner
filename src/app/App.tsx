import React, { FunctionComponent } from 'react'
import { Provider } from 'react-redux'
import {Switch, BrowserRouter as Router, Route } from "react-router-dom"
import { Store } from 'redux'
import { Menu } from '../menu/containers/Menu'
import './App.less'

export const App: FunctionComponent<{store: Store<RootState>}> = ( props ) => (
	<Provider store={props.store}>
	<Router>
		<div>
			<Switch>
				<Route exact path="/" component={Menu} />
			</Switch>
		</div>
	</Router>
	</Provider>
)
