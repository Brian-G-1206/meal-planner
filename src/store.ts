import { configureStore } from "@reduxjs/toolkit"
import { reducer as menuReducer } from './menu/reducer'

const initialState: RootState = {
	menu: {
		dinnerOptions: [],
	}
}
export const store = configureStore({
	reducer: {
		menu: menuReducer
	},
	devTools: true,
	preloadedState: initialState,
})