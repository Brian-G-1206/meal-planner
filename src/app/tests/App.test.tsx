import { shallow, ShallowWrapper } from 'enzyme'
import { App } from '../App'
import { store } from '../../store'


describe('src/app/App', () => {
	it('should match snapshot', () => {
		const Subject: ShallowWrapper = shallow(<App store={store} />)
		expect(Subject).toMatchSnapshot()
	  })
	  
})
