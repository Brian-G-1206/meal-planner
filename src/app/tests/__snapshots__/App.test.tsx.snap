// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/app/App should match snapshot 1`] = `ShallowWrapper {}`;
